CMakeLists.txt

```diff
@@ -75,6 +75,8 @@ find_program(THRIFT_COMPILER thrift)
 find_path(THRIFT_INCLUDE_DIR thrift)
 find_library(THRIFT_LIBRARY NAMES thrift)
 
+find_package(Boost REQUIRED COMPONENTS system thread)
```

Comment:
For `evpp`, boost is not a required dependency. It is only an optional dependency.

Reply:
The build requires it though. Should this move to the component level?

Reply:
I think that's will be better.